### PR TITLE
Nuke: ayon server settings improvements

### PR DESCRIPTION
**Pull request by jakubjezek001**
## Changelog Description
Nuke settings were not aligned with OpenPype settings. Also labels needed to be improved.

## Additional info
- imroving consistency of key names

## Testing notes:
1. all should be working as it was before
2. create preset in `ayon+settings://nuke:0.1.4/imageio/nodes/override_nodes?project=SyncSketchTesting` and notice there is `Subsets` input

### CODE REVIEW

**Review by moonyuet — APPROVED** [review]
LGTM

File: openpype/settings/ayon_settings.py [hunk 1]
```diff
@@ -819,9 +819,45 @@ def _convert_nuke_project_settings(ayon_settings, output):
...
+
+    # regex inputs
+    regex_inputs = ayon_imageio.get("regex_inputs")
+    if regex_inputs:
+        ayon_imageio.pop("regex_inputs")
+        ayon_imageio["regexInputs"] = regex_inputs
+
+    # nodes
+    ayon_imageio_nodes = ayon_imageio["nodes"]
+    if ayon_imageio_nodes.get("required_nodes"):
```
Comment (iLLiCiTiT) [inline openpype/settings/ayon_settings.py:846]:
```suggestion
    if "required_nodes" in ayon_imageio_nodes:
```

File: openpype/settings/ayon_settings.py [hunk 2]
```diff
@@ -819,9 +819,45 @@ def _convert_nuke_project_settings(ayon_settings, output):
...
+        ayon_imageio_nodes["requiredNodes"] = (
+            ayon_imageio_nodes.pop("required_nodes"))
+    if ayon_imageio_nodes.get("override_nodes"):
```
Comment (iLLiCiTiT) [inline openpype/settings/ayon_settings.py:849]:
```suggestion
    if "override_nodes" in ayon_imageio_nodes:
```

File: openpype/settings/ayon_settings.py [hunk 3]
```diff
@@ -819,9 +819,45 @@ def _convert_nuke_project_settings(ayon_settings, output):
...
+        ayon_imageio_nodes["overrideNodes"] = (
+            ayon_imageio_nodes.pop("override_nodes"))
+
+    for item in ayon_imageio_nodes["requiredNodes"]:
+        if item.get("nuke_node_class"):
+            item["nukeNodeClass"] = item["nuke_node_class"]
```
Comment (iLLiCiTiT) [inline openpype/settings/ayon_settings.py:855]:
```suggestion
        if "nuke_node_class" in item:
            item["nukeNodeClass"] = item.pop("nuke_node_class")
```

File: openpype/settings/ayon_settings.py [hunk 4]
```diff
@@ -819,9 +819,45 @@ def _convert_nuke_project_settings(ayon_settings, output):
...
         item["knobs"] = _convert_nuke_knobs(item["knobs"])
-    for item in ayon_imageio["nodes"]["overrideNodes"]:
+
+    for item in ayon_imageio_nodes["overrideNodes"]:
+        if item.get("nuke_node_class"):
+            item["nukeNodeClass"] = item["nuke_node_class"]
```
Comment (iLLiCiTiT) [inline openpype/settings/ayon_settings.py:860]:
```suggestion
        if "nuke_node_class" in item:
            item["nukeNodeClass"] = item.pop("nuke_node_class")
```

File: openpype/settings/ayon_settings.py (hunk 2 above)
Comment (iLLiCiTiT) [inline openpype/settings/ayon_settings.py:849]:
```suggestion
    if "override_nodes" in ayon_imageio_nodes:
```

File: openpype/settings/ayon_settings.py [hunk 5]
```diff
@@ -819,9 +819,45 @@ def _convert_nuke_project_settings(ayon_settings, output):
...
+            src in imageio_workfile
+            and dst not in imageio_workfile
+        ):
+            imageio_workfile[dst] = imageio_workfile.pop(src)
+
+    # regex inputs
+    regex_inputs = ayon_imageio.get("regex_inputs")
+    if regex_inputs:
+        ayon_imageio.pop("regex_inputs")
+        ayon_imageio["regexInputs"] = regex_inputs
```
Comment (iLLiCiTiT) [inline openpype/settings/ayon_settings.py:842]:
```suggestion
    if "regex_inputs" in ayon_imageio:
        ayon_imageio["regexInputs"] = ayon_imageio.pop("regex_inputs")
```

**Review by iLLiCiTiT — CHANGES_REQUESTED** [review]
Please look at the suggestions.

File: openpype/settings/ayon_settings.py (hunk 2 above)
Comment (jakubjezek001) [reply openpype/settings/ayon_settings.py]: implemented

**Comment by jakubjezek001** [comment]
@iLLiCiTiT all of your suggestions were committed, thanks to them ;)

**Review by iLLiCiTiT — APPROVED** [review]
Looks good.

**Review by 64qam — APPROVED** [review]

**Review by 64qam — APPROVED** [review]

**Pull request merged**